mona_openai/util/math.py

```
@@ -0,0 +1,11 @@
+"""
+Utility module for math-related functionality.
+"""
+
+
+def get_factors(iter_1, iter_2):
+    """
+    For two given iterables of numbers, returns a tuple of the factors
+    between each pair of numbers from the two lists.
+    """
+    return tuple(x / y for x, y in zip(iter_1, iter_2))
```

1. What if one of the numbers in iter_2 is 0?
2. What if one of the values is not a number?
3. What if the given iterable has a different length?

---

1. Good catch. Let's add a None in that case
2. Then we'll throw an exception which I'm good with as expected behavior. 
3. Let's throw an exception in that case as well. 

Good catches across the board :)